TickSpec.XunitWiring/TickSpec.XunitWiring.fsproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+<!--  <Import Project="../netfx.props" />-->
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;net452</TargetFrameworks>
+    <RootNamespace>TickSpec.Xunit</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="FeatureSetup.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\TickSpec\TickSpec.fsproj" />
+    <PackageReference Include="Xunit" Version="2.4.1" />
```

Comment:
ideally this would be the lowest 2.x version that works so as not to force updates on people

Reply:
Yes, got it, but look, it was 2.4.0 before this PR, so I just put it back to that now, should be fine I hope.